packages/components/src/custom-gradient-picker/index.native.js

```diff
@@ -25,14 +25,15 @@ function CustomGradientPicker( { setColor, currentValue, isGradientColor } ) {
 	const [ currentColor, setCurrentColor ] = useState( currentValue );
 
 	const { getGradientType, gradients, gradientOptions } = colorsUtils;
-	const gradientAST = getGradientAstWithDefault( currentColor );
+	const gradientWithDefault = getGradientAstWithDefault( currentColor );
```

Comment:
If we wanted to avoid the additional new lines, de-structuring this might work as it appears we do not use `gradientWithDefault` directly anywhere. 

```js
const { gradientAST } = getGradientAstWithDefault( currentColor );
```

Reply:
Definitely, I'm all for removing additional lines 😄 I'll update it!

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/49917/commits/4a7a4c0405b9236e8c975fb3d92cfe04cdf1a57f, thanks for the suggestion!